examples/GateNoiseModels.ipynb

```diff
@@ -319,11 +319,11 @@
     "    \n",
     "    print(\"\\r{}/{}, \".format(jj, len(lengths)), end=\"\")\n",
     "\n",
-    "    \n",
     "    p = Program(X(0))\n",
+    "    mem = p.declare(\"ro\")\n",
```

Comment:
is there some reason you're not naming this variable `ro`?

Reply:
🤷‍♂